README.md

```diff
@@ -61,15 +61,15 @@ int decryptedLen = cis.read(decryptedData, 0, 1024);
 ### Configuration
 Currently, two ciphers are supported: JceCipher and OpensslCipher, you can configure which cipher to use as follows:
 
-    $ java -Dchimera.crypto.cipher.classes=com.intel.chimera.crypto.OpensslCipher Sample
-    $ java -Dchimera.crypto.cipher.classes=com.intel.chimera.crypto.JceCipher Sample
+    $ java -Dchimera.crypto.cipher.classes=org.apache.commons.crypto.crypto.OpensslCipher Sample
```

Comment:
There is "crypto.crypto" in the package name which is a strange name pattern. Create another JIRA to address the naming issue.